src/views/activityBar/projectView/ProjectDataProvider.ts

```
@@ -0,0 +1,274 @@
+  private triggerLoadingState = (entity: string) => {
+    this.items = [LOADING_TREE_ITEMS.get(entity)!];
+    this._onDidChangeTreeData.fire(undefined);
+  };
+
+  /**
+   * Retrieves the singleton instance of ProjectDataProvider.
+   *
+   * @returns {ProjectDataProvider} The singleton instance.
+   */
+  public static getInstance(): ProjectDataProvider {
+    if (!this.instance) {
```

Biome (1.9.4)

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)